tests/unit/components/playback/controls/SettingsFormVuex.spec.js

```diff
@@ -0,0 +1,203 @@
+  });
+  test("Given that the SettingsForm is loaded with Vuex store, When there is no data in Vuex, Then <empty> is set in the Input of Customer ID and User ID", () => {
+    wrapper = mount(ComponentToTest, {
+      store,
+      localVue,
+    });
+    wrapper.vm.onCancelAddCustomerId(); // Complete Code Coverage
+    expect(
+      wrapper.find("#input-dropdown-widget-cust-").element.value
+    ).toStrictEqual("");
+    expect(
+      wrapper.find("#input-dropdown-widget-user-").element.value
+    ).toStrictEqual("");
+  });
+  test("Given that the SettingsForm is loaded with Vuex store, When there is no data in the user_ids, Customer Index/User Index is not set in the Vuex, Then <empty> is set in the Input of  Customer ID/User ID", () => {
+    store.commit("settings/set_customer_account_ids", array_of_special_userids);
+    wrapper = mount(ComponentToTest, {
+      store,
+      localVue,
+    });
+    expect(
+      wrapper.find("#input-dropdown-widget-cust-").element.value
+    ).toStrictEqual("");
+    expect(
+      wrapper.find("#input-dropdown-widget-user-").element.value
+    ).toStrictEqual("");
+  });
+  test("Given that the SettingsForm is loaded with Vuex with different set of null user_ids from store, When there is no data in the user_ids Customer Index/User Index is not set in the Vuex, Then <empty> is set in the Input of  Customer ID/User ID", async () => {
+    store.commit(
+      "settings/set_customer_account_ids",
+      array_of_special_customerid_null_userids
+    );
+    store.commit("settings/set_customer_index", 0);
+    wrapper = mount(ComponentToTest, {
+      store,
+      localVue,
+    });
+
+    expect(
+      wrapper.find("#input-dropdown-widget-cust-").element.value
+    ).toStrictEqual("");
+    expect(
+      wrapper.find("#input-dropdown-widget-user-").element.value
+    ).toStrictEqual("");
+  });
+  test("Given that the SettingsForm is loaded with Vuex with 'Customer account -1' 'User account -1', When user decides to modify Customer ID to 'Customer account -2' 'Lab User -1', Then validate 'Save Changes' updates validate Vuex store with is updated index values", async () => {
+    store.commit("settings/set_customer_account_ids", array_of_customerids);
+    store.commit("settings/set_customer_index", 0);
+    store.commit("settings/set_user_index", 0);
+
+    wrapper = mount(ComponentToTest, {
+      store,
+      localVue,
+    });
+    await wrapper.vm.$nextTick(); // wait for update
+
+    await wrapper
+      .find("#input-dropdown-widget-cust-")
+      .setValue("Customer account -2");
+    await wrapper.vm.$nextTick(); // wait for update
+    await wrapper.find("#input-dropdown-widget-user-").setValue("Lab User -1");
+    await wrapper.vm.$nextTick(); // wait for update
+
+    await wrapper.vm.$nextTick(); // wait for update
+
+    wrapper.vm.savechanges();
```

Comment:
Whenever possible, we should trigger an action through the DOM instead of `wrapper.vm`. This mimics the way a user could interact with the system, which makes the test more representative of the user experience, and it also avoids testing the internals of the component (like the name of the method `savechanges`) which means we can refactor our code (i.e. change method names) without breaking tests. Can you instead click on the Save Changes button in the DOM?

Reply:
Also, `savechanges` shouldn't be a single word, it should be `save_changes`. Please update in the component itself

Reply:
Corrected with `savechanges` to `save_changes` 
DOM function is modified to 

```
const save_changes = wrapper.find(".span__settings-tool-tip-save-btn-txt-enable");
    await save_changes.trigger("click");
```